pyvista/core/grid.py

```diff
@@ -685,7 +694,7 @@ def points(self) -> NumpyArray[float]:
         return points
 
     @points.setter
-    def points(self, points):  # numpydoc ignore=PR01
+    def points(self: ImageData, points: Never) -> Never:  # numpydoc ignore=PR01
```

Comment:
I did not know about the type `Never` . Please wait as I am learning its usage.

Reply:
I think it has an alias `NoReturn` which may be more explicit.

We can also remove it and hint the actual types, i.e. make them have the same signature as `PointSet.points` setter